abcclassroom/__main__.py

```diff
@@ -22,6 +22,11 @@
 from .notebook import split_notebook
 from . import github as GH
 from .utils import copytree, P, input_editor
+from .dirsetup import directory_setup
```

Comment:
@nkorinek let's follow @kcranston suggestion and call this module quickstart rather than dirsetup.

i'm also getting confused because there is dirsetup, directory_Setup etc... dir_setup. eeks too many similar names!! :)

Reply:
Ya, the names are unfortunate for sure. I'll fix that with the rename to quickstart.

Reply:
thanks @nkorinek ping us with any questions or we can chat more on slack if you want too!!